Source/hydro/ppm_nd.F90

```diff
@@ -145,39 +143,39 @@ subroutine ppm_type1(s, s_lo, s_hi, ncomp, n, &
     real(rt), intent(in) :: dx(3)
 
     ! local
-    integer i,j,k
+    integer i, j, k
 
     real(rt)         dsl, dsr, dsc
 
     ! s_{\ib,+}, s_{\ib,-}
     real(rt)         :: sm, sp
 
     ! \delta s_{\ib}^{vL}
-    real(rt)        , pointer :: dsvl(:,:)
+    real(rt)        , pointer :: dsvl(:,:,:)
     real(rt)         :: dsvlm, dsvl0, dsvlp
 
     ! s_{i+\half}^{H.O.}
-    real(rt)        , pointer :: sedge(:,:)
+    real(rt)        , pointer :: sedge(:,:,:)
 
-#ifndef AMREX_USE_CUDA    
+#ifndef AMREX_USE_CUDA
     if (ppm_type .ne. 1) &
          call amrex_error("Should have ppm_type = 1 in ppm_type1")
 
-    if (s_lo(1) .gt. ilo1-3 .or. s_hi(1) .lt. ihi1+3) then 
+    if (s_lo(1) .gt. lo(1)-3 .or. s_hi(1) .lt. hi(1)+3) then
          call amrex_error("Need more ghost cells on array in ppm_type1")
     end if
 
 #if (AMREX_SPACEDIM >= 2)
-    if (s_lo(2) .gt. ilo2-3 .or. s_hi(2) .lt. ihi2+3) then
+    if (s_lo(2) .gt. lo(2)-3 .or. s_hi(2) .lt. hi(2)+3) then
          call amrex_error("Need more ghost cells on array in ppm_type1")
```

Comment:
I suppose we should now also add an error check for the z dimension

Reply:
yes.  Although this will be modified in a later PR anyway, since lo and hi will have a different meaning then.